AdyenCard/Components/Card/CardComponent.swift

```diff
@@ -308,3 +302,14 @@ private extension CardComponent.Configuration {
         )
     }
 }
+
+// MARK: - SubmitCustomizable
+
+extension CardComponent: SubmitCustomizable {
+
+    public func submit() {
+        guard !configuration.showSubmitButton else { return }
```

Comment:
Why do we need this check?
If a merchant wants to call submit themselves even when hiding the button why are we preventing them?

Reply:
Done